Header          header
uint32          week
float64         timeOfWeek
uint32          insStatus
uint32          hdwStatus
float32[4]      qe2b
float32[3]      ve
float64[3]      ecef
